core/input/input.cpp

```diff
@@ -1365,10 +1365,12 @@ void Input::parse_mapping(String p_mapping) {
 
 		JoyButton output_button = _get_output_button(output);
 		JoyAxis output_axis = _get_output_axis(output);
-		ERR_CONTINUE_MSG(output_button == JoyButton::INVALID && output_axis == JoyAxis::INVALID,
-				vformat("Unrecognized output string \"%s\" in mapping:\n%s", output, p_mapping));
-		ERR_CONTINUE_MSG(output_button != JoyButton::INVALID && output_axis != JoyAxis::INVALID,
-				vformat("Output string \"%s\" matched both button and axis in mapping:\n%s", output, p_mapping));
```

Comment:
I'm not too familiar with the code but isn't this one still one situation which should never happen? If so, keep it as an error message makes sense to me.